scl/slack/slack.conf

```diff
@@ -21,8 +21,6 @@
 #############################################################################
 
 @requires http
-@requires json-plugin
-@requires basicfuncs
```

Comment:
Please do not remove this; either move them into the block like http or leave them here.

Reply:
I have separated this to the last patch, which can be dropped depending on the discussion we have on making json-plugin mandatory. But I'm resolving this conversation.